.circleci/config.yml

```diff
@@ -118,13 +118,48 @@ commands:
       - run:
           name: Deploy 3scale from amp-eval template without imagestreams
           command: |
+            WILDCARD_DOMAIN="lvh.me"
             ruby -ryaml -rjson -e 'puts YAML.load(ARGF).tap{|t| t["objects"].reject!{|o| o["kind"]=="ImageStream"}}.to_json' pkg/3scale/amp/auto-generated-templates/amp/amp-eval.yml | \
-              oc new-app -f- --param WILDCARD_DOMAIN=lvh.me --param AMP_RELEASE=master
+              oc new-app -f- --param WILDCARD_DOMAIN=${WILDCARD_DOMAIN} --param AMP_RELEASE=master
             oc wait --for=condition=available --timeout=-1s $(oc get dc --output=name)
 
             oc get events | egrep ' Failed ' || :
             oc get events -o json | jq '[.items[] | select(.reason == "Failed") | debug ] | length == 0'  --exit-status
           no_output_timeout: 30m
+      - check-3scale-templates-deployed-routes
+
+  check-3scale-templates-deployed-routes:
+    parameters:
```

Comment:
any job using this wildcard_domain parameter?

Reply:
I implemented it as a parameter to make the command more reusable/configurable. 
In the command itself the value of the parameter is expanded